packages/framework/src/Component/Elasticsearch/AbstractIndex.php

```diff
@@ -19,6 +19,32 @@ abstract class AbstractIndex
      */
     abstract public function getTotalCount(int $domainId): int;
 
+    /**
+     * This method is not abstract because you can implement the index without the export of only changed rows
+     * In that case `ElasticsearchChangedDataExportCommand` will not export such index at all
+     *
+     * @param int $domainId
+     * @return int
+     */
+    public function getChangedCount(int $domainId): int
+    {
+        return 0;
```

Comment:
Thank you for this. I agree it could be confusing. 
I introduced the interface the indices supporting "changed" functionality has to implement.
The behavior is more or less the same when an index does not implement this interface (index is skipped), but from the output, it's now clear why.

Is it something you had in mind? implemented in 1129902d0 and 3fed5a3d3

Reply:
That is nice. Please update a cookbook too.

Reply:
Updated in f5da54518